Dockerfile

```diff
@@ -1,21 +1,17 @@
 # download kubectl
-FROM golang:1.21.3-alpine as kubectl
-RUN apk add --no-cache curl
+FROM golang:1.21.3 as kubectl
```

Comment:
Since none of these stages are building code any more, you could drop the `golang` images altogether